addons/common/fnc_isTurnedOut.sqf

```diff
@@ -35,21 +34,20 @@ private _role = toLower (_unit call CBA_fnc_vehicleRole);
 private _isFFV = fullCrew [_vehicle, "turret"] findIf {_x select 0 == _unit && {_x select 4}} != -1;
 
 // special condition for RHS car windows
-private _windowsOpen = false;
 if (_vehicle isKindOf "rhsusf_m1025_w" && {_role in ["driver", "turret"]}) then {
-    // will override _isFFV if window is closed
+    //will override _isFFV if window is closed
     _isFFV = switch (_vehicle getCargoIndex _unit) do {
```

Comment:
```
_vehicle doorPhase
(["ani_window_1","ani_window_2", "ani_window_4"] param [(_vehicle getCargoIndex _unit) + 1, "ani_window_3"])
> 0
```
magic

Reply:
i did the same
```
private _cargoIndex = (_vehicle getCargoIndex _unit) min 2;
private _window = ['ani_window_1', 'ani_window_2', 'ani_window_4', 'ani_window_3'] select (_cargoIndex + 1);
```
@commy2 didnt liked it 😄